src/runtime_src/core/include/ert.h

```diff
@@ -569,10 +574,10 @@ ert_fill_copybo_cmd(struct ert_start_copybo_cmd *pkt, uint32_t src_bo,
   pkt->cu_mask[1] = 0;
   pkt->cu_mask[2] = 0;
   pkt->cu_mask[3] = 0;
-  pkt->src_addr_lo = src_offset;
+  pkt->src_addr_lo = (uint32_t)src_offset;
```

Comment:
Should this code be instead: (uint32_t)(src_offset & 0xFFFFFFFF).  Shouldn't be an issue, but it does prevent a potential 1's compliment  issue.